.github/workflows/backend_ci.yml

```
@@ -0,0 +1,157 @@
+              if: ${{ steps.test.outputs.test_status == 'failure' }}
+              run: |
+                  echo "❌ Tests failed. Stopping workflow."
+                  exit 1
+
+    build:
+        needs: test
+        runs-on: ubuntu-latest
+        timeout-minutes: 15
+
+        steps:
+            - uses: actions/checkout@v3
```

actionlint (1.7.7)

the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)